### PR TITLE
Refactor input

### PR DESCRIPTION
**Pull request by janisozaur**
This is fairly small change that converts parts of input state to have more interfaces rather than changing said states directly.

### CODE REVIEW

File: src/openrct2/world/tile_inspector.c [hunk 1]
```diff
@@ -24,6 +24,10 @@
 #include "map.h"
 #include "tile_inspector.h"
 
+uint32 windowTileInspectorTileX;
+uint32 windowTileInspectorTileY;
+sint32 windowTileInspectorElementCount = 0;
```
Comment (IntelOrca) [inline src/openrct2/world/tile_inspector.c]: `_windowTileInspectorElementCount`

File: src/openrct2/input.h [hunk 2]
```diff
@@ -70,14 +49,35 @@ enum MOUSE_STATE {
...
+
+#include "interface/window.h"
+
 typedef struct widget_ref {
 	rct_windowclass window_classification;
 	rct_windownumber window_number;
 	rct_widgetindex widget_index;
 } widget_ref;
 
-extern uint8 gInputState;
```
Comment (duncanspumpkin) [inline src/openrct2/input.h]: These should be changed to `_inputState` as they are no longer global.

Comment (janisozaur) [inline src/openrct2/input.h]: done

File: src/openrct2/world/tile_inspector.c (hunk 1 above)
Comment (janisozaur) [inline src/openrct2/world/tile_inspector.c]: removed them for now, I think they got in by mistake

**Pull request merged**